app/main.py

```diff
@@ -1 +1,38 @@
-# write your code here
+from __future__ import annotations
+
+
+class Animal:
+    alive = []
```

Comment:
The `alive` list is a class variable, meaning it is shared across all instances of `Animal`. This can lead to unexpected behavior if not managed carefully, especially in a multi-threaded environment or if instances are deleted without being removed from the list.